mods/snowyPowerup.py

```diff
@@ -179,8 +183,18 @@ def handleMessage(self,m):
                         self.handleMessage(bs.DieMessage())
                     elif self.powerupType == 'portal':
                         t = bsSpaz.gPowerupWearOffTime
-                        self.port = Portal.Portal(position1 = node.position,r = 0.9,color = (random.random(),random.random(),random.random()),activity = bs.getActivity())
-                        bs.gameTimer(t,bs.Call(self.port.delete))
+                        if Portal.currentnum < Portal.maxportals :
+                            Portal.currentnum += 1
+                            if self.node.position in Portal.lastpos :
+                                self.port = Portal.Portal(position1 = None,r = 0.9,color = (random.random(),random.random(),random.random()),activity = bs.getActivity())
+                                bs.gameTimer(t,bs.Call(self.delpor))
+                            else :
+                                m = self.node.position
+                                Portal.lastpos.append(m)
+                                self.port = Portal.Portal(position1 = self.node.position,r = 0.9,color = (random.random(),random.random(),random.random()),activity = bs.getActivity())
+                                bs.gameTimer(t,bs.Call(self.delpor))
+                        else :
+                            bs.screenMessage('Only '+str(Portal.maxportals)+' pairs of portals are allowed at a time.')
```

Comment:
We definitely don't need a reminder every time a portal powerup is collected past three.  I think people can figure it out by themselves.  Also, it won't happen too, too often, so it's better to just not interrupt game-flow in my opinion.

Reply:
Hmm, k I remove now.

Reply:
I did remove the screen message but the pull request yet shows change required, why is this so?